file_paths.md

```diff
@@ -281,8 +280,10 @@ create a `puppet` user or group.
     C:\Program Files\Puppet Labs\Puppet\pxp-agent *
         bin *
             pxp-agent.exe *
+            lib*.dll *
```

Comment:
Might want to note what these are for in the comments

```
lib*.dll *                    # gcc, mingw, boost libraries
```

Could we also add `libeay32.dll` to the OpenSSL section with `ssleay32.dll`?